src/components/YourEmail/YourEmail.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import {Field, Form} from 'react-final-form'
+import './YourEmail.scss'
+
+interface IProps {
+    submitBtn: any;
+    input: string;
+    setInput: any;
+    validateEmail: any
+}
+
+const YourEmail:React.FC<IProps> = ({submitBtn, input, validateEmail, setInput}) => {
```

Comment:
Imho this component should be called just "Email" or "EmailField", because that's what it is essentially